currency-converter/src/components/MainHeading.tsx

```diff
@@ -1,3 +1,6 @@
-export default function MainHeading({ name }: { name: string }) {
-  return <h1 className="text-3xl text-center font-bold">{name}</h1>;
+interface Name {
```

Comment:
interface name should be more meaningful such that by just reading interface name, we should know where its used. Usually a convention is followed for naming interface, we just append capital I before component's name where it'll be used. So it'll be "IMainHeading", so any other developer will know that this interface is used for MainHeading component.